scripts/jenkins/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+srcdir=`dirname $0`
+test -z "$srcdir" && srcdir=.
+. ${srcdir}/common.bash
+
+jenkins_setup
+
+go get -v -u github.com/jstemmer/go-junit-report
+go get -v -u github.com/t-yuki/gocover-cobertura
+go get -v -t ./...
+
+export COV_FILE="build/coverage/coverage.cov"
+export OUT_FILE="build/test-report.out"
+mkdir -p build/coverage
+
+(go test -race ./... -v -coverprofile="${COV_FILE}" -coverpkg=go.elastic.co/apm/... 2>&1 | tee ${OUT_FILE}) || echo -e "\033[31;49mTests FAILED\033[0m"
```

Comment:
still running coverage here, this should just be `go test -v -race ./...`, and the cover/gocover-cobertura bits below removed